contracts/ERC721RExample.sol

```diff
@@ -122,6 +127,8 @@ contract ERC721RExample is ERC721A, Ownable {
 
     function toggleRefundCountdown() public onlyOwner {
         refundEndTime = block.timestamp + refundPeriod;
+        console.log("Refund counted started to block");
```

Comment:
if we can remove logs from the pr please :)

Reply:
Yup